### PR TITLE
docs(npm): update docs on publishing npm packages, update org name (github + npm)

### PR DESCRIPTION
**Pull request by ruxandrafed**
## Overview

- Update all links & references to other repositories to use `telus` as the organization name (instead of `telusdigital`).
- Update all links & references to npm packages to use `telus` as the scope/org name (instead of `telusdigital`).
- Update docs on publishing npm packages with current recommended approach (using [@telus/create-library][create-library])

---

#### Meta

> Please read and confirm each of the following:

- [x] this topic was discussed in the [Technology Forum][technology-forum] _(ignore if the pull request represents small changes)_
- [x] provided a descriptive topic and overview of contribution
- [x] documentation format follows the [topic template][template]
- [x] fork is up to date _(Hint: ["Syncing a Fork"][guide-forks])_
- [x] "work in progress" commits are squashed _(Hint: ["Squashing Commits"][guide-squash])_
- [x] commits follow the [Conventional ChangeLog][conventional-changelog] format
- [x] no sensitive content included, such as:
  - content considered competitive intelligence
  - security & privacy policy violating content
  - keys, tokens or credentials

[create-library]: https://github.com/telus/create-library
[template]: https://github.com/telus/reference-architecture/blob/master/.template.md
[conventional-changelog]: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular
[guide-forks]: https://help.github.com/articles/syncing-a-fork/
[guide-squash]: https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History
[technology-forum]: https://github.com/telus/technology-forum]

### CODE REVIEW

File: development/libraries.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ Any library should:
 - Be explicit about dependencies
 - Include documentation with example usage
 - Follow the versioning strategy defined [here](versioning.md)
-- Include [automated tests](https://github.com/telusdigital/reference-architecture/tree/master/testing)
-- Have clear [contribution guidelines](https://github.com/telusdigital/reference-architecture/blob/master/.github/CONTRIBUTING.md)
-- Use the [npm library starter Kit](https://github.com/telusdigital/npm-library-starter-kit) as a starting point.
+- Include [automated tests](https://github.com/telus/reference-architecture/tree/master/testing)
+- Have clear [contribution guidelines](https://github.com/telus/reference-architecture/blob/master/.github/CONTRIBUTING.md)
+- Use the [npm library starter kit (@telus/create-library)](https://github.com/telus/create-library) as a starting point.
```
Comment (kspaans) [inline development/libraries.md]: This should point to `create-library` now! :)

Comment (ruxandrafed) [reply development/libraries.md]: @kspaans Help me figure out what I missed 'cause I'm blind 😄 I updated the npm library starter kit link 🤔

File: development/starter-kits.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ They have been developed in a collaborative partnership with the following teams
 
 ### Ownership and updating
 
-No one team owns the starter kits. They are worked on collaboratively, with an open-source pull-request model. The standards are managed through the weekly [Architecture Forum](https://github.com/telusdigital/architecture-forum), on [#TechMondays](https://telusdigital.atlassian.net/wiki/display/techmondays/Schedule). If you are looking for a new feature to be added, you should join the forum and open an issue, although this is not required to submit a pull request.
+No one team owns the starter kits. They are worked on collaboratively, with an open-source pull-request model. The standards are managed through the weekly [Architecture Forum](https://github.com/telus/architecture-forum), on [#TechMondays](https://telusdigital.atlassian.net/wiki/display/techmondays/Schedule). If you are looking for a new feature to be added, you should join the forum and open an issue, although this is not required to submit a pull request.
```
Comment (kspaans) [inline development/starter-kits.md]: Rename to Tech Forum, and remove the refrence to tech mondays: the new tech forum README specifies the schedule.

**Pull request merged**